src/nidigital/examples/nidigital_self_cal_and_test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```

Comment:
I'd like to review this in its own PR please.

Reply:
Made the example a do_nothing example. We need _a_ file here that passes flake8 to make the build process happy.